src/applications/disability-benefits/all-claims/constants.js

```diff
@@ -244,9 +244,3 @@ export const ATTACHMENT_KEYS = [
 export const LOWERED_DISABILITY_DESCRIPTIONS = Object.values(
   disabilityLabels,
 ).map(v => v.toLowerCase());
-
-// This comes straight from EVSS but isn't documented in Swagger because
-// it's only used on disabilities that don't exist in the mapped list.
-// Note: the right single quote (’) is intentional - apostrophes are not
-//       allowed.
-export const EVSS_DISABILITY_NAME_REGEX = /^([a-zA-Z0-9\-‘.,/()]([a-zA-Z0-9\-’,. ])?)+$/;
```

Comment:
Why isn't this needed anymore?

Reply:
That's basically the point of the PR 😛 
We're not going to validate the new disability names on the front end anymore. So we won't need the regex we used in the validator. Does this need more explanation?

Reply:
...we are going to validate the length though, because it would be silly to truncate someone's response in vets-api.